api_app/analyzers_manager/views.py

```diff
@@ -70,3 +70,11 @@ def get_object(self, job_id, analyzer_name) -> AnalyzerReport:
     def _post_kill(self, report):
         # clean up job
         analyzers_controller.job_cleanup(report.job)
+
+    def _start_retry(self, report: AnalyzerReport):
+        analyzers_to_execute = [report.analyzer_name]
+        report.job.status = "running"  # update job status to running
+        report.job.save(update_fields=["status"])
+        analyzers_controller.start_analyzers(
+            report.job.id, analyzers_to_execute, report.runtime_configuration
```

Comment:
The `report.runtime_configuration` won't work like this.

The `runtime_configuration` stored in `AbstractReport` stores only the key-value pairs but the `start_analyzers` method expects `runtime_configuration` in the format: `{"ANALYZER_NAME": {"key1": "value1", "key2": "value2"} }`